ynput/OpenPype
# Resolve: after launch automatization fixes

## Changelog Description
Workfile is no correctly created and aligned witch actual project. Also the launching mechanism is now fixed so even no workfile had been saved yet it will open OpenPype menu automatically.

## Additional info
Added testing script for startup.py  - it showing only if openpype has been run from code. 

## Testing notes:
1. make sure no workfile version is created in your task where resolve is launched. 
2. make sure `project_settings/resolve/launch_openpype_menu_on_start` in project settings are activated.
3. Openpype menu should popup after Resolve is active. 
4. Save as new workfile version from workfile tool and see that all is created and activated. 
5. Close Resolve and start it again. 
6. see that Resolve finds latest version and opens it with OpenPype menu
2. follow this step
